### ISSUE TITLE
Get Historical Data - Service unavailable

### ISSUE THREAD

**Issue by BarnumD** [Drakkar-Software/OctoBot#2559]
## Step 1: Have you search for this issue before posting it?

If you have discovered a bug in the bot, please [search our issue tracker](https://github.com/Drakkar-Software/OctoBot/issues?q=is%3Aissue). 
If it hasn't been reported, please create a new issue.

## Step 2: Describe your environment
  * OS : Linux Docker image 1.0.8

## Step 3: Describe the problem:

On the /backtesting page, clicking Get Historical Data; Receive the following error:
Error: Failed to load_symbol_markets: ExchangeNotAvailable on binance GET https://api.binance.com/api/v3/exchangeInfo 451 { "code": 0, "msg": "Service unavailable from a restricted location according to 'b. Eligibility' in https://www.binance.com/en/terms. Please contact customer service if you believe you received this message in error." } (FailedRequest). More details in logs.

**Expected behavior**
Download data

### Steps to reproduce:

  1. /backtesting page
  2. Click get historical data
  
### Observed Results:

  * I am based in the U.S. so that may be a reason for the error.
  * Under /accounts, I have no binance exchange.  I only have Kraken.

**Comment by Herklos** [comment]
Hi @BarnumD, it seems that you tried to access regular binance with a US IP. It's not possible due to Binance.com terms, you should use Binance US exchange

**Comment by BarnumD** [comment]
Yes but I had the Binance exchange completely removed when this occurred.  I only had kraken exchange enabled.    ---- On Sun, 17 Mar 2024 10:55:16 -0400  ***@***.***> wrote ----  
Hi @BarnumD, it seems that you tried to access regular binance with a US IP. It's not possible due to Binance.com terms, you should use Binance US exchange
